requirements/testing.txt

```diff
@@ -7,7 +7,7 @@
 #
 -r development.txt
 -r integration.txt
--e file:.
+-e file:///mnt/c/temp/superset_numexp
```

Comment:
remove this and replace it with `.` like it was before

Reply:
fixed it back to .